sdks/python/apache_beam/runners/interactive/examples/Run Beam SQL with beam_sql magic.ipynb

```diff
@@ -0,0 +1,548 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bcf28852",
+   "metadata": {},
+   "source": [
+    "## Query#1 - A simple static query\n",
+    "`beam_sql` is an IPython [custom magic](https://ipython.readthedocs.io/en/stable/config/custommagics.html). If you're not familiar with magics, here are some [built-in examples](https://ipython.readthedocs.io/en/stable/interactive/magics.html).\n",
+    "\n",
+    "The `beam_sql` magic can be used as either a line magic or a cell magic.\n",
+    "We can check its usage by running:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5777a84e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%beam_sql -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20f88c5d",
+   "metadata": {},
+   "source": [
+    "We can run a simple SQL query (in Apache Calcite SQL [syntax](https://beam.apache.org/documentation/dsls/sql/calcite/query-syntax/)) to create a [schema-aware PCollection](https://beam.apache.org/documentation/programming-guide/#schemas) from static values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49c70de3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%beam_sql -o query1_data\n",
+    "SELECT CAST(5 AS INT) AS `id`, CAST('foo' AS VARCHAR) AS `str`, CAST(3.14 AS DOUBLE) AS `flt`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a1c5e9e",
+   "metadata": {},
+   "source": [
+    "The `beam_sql` magic shows you the result of the SQL query.\n",
+    "\n",
+    "It also creates and outputs a PCollection named `query1_data` with element_type like `BeamSchema_...(id: int32, str: str)`.\n",
+    "\n",
+    "Note that we have **not** explicitly created a Beam pipeline. We get a PCollection because the `beam_sql` magic always **implicitly creates** a pipeline to execute your SQL query. To hold the elements with each field's type info, Beam automatically creates a schema as the element_type for the created PCollection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05d2b688",
+   "metadata": {},
+   "source": [
+    "To introspect the data again with more knobs, we can use `show`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89220b13",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from apache_beam.runners.interactive import interactive_beam as ib\n",
+    "ib.show(query1_data)\n",
+    "# Uncomment below to set more args.\n",
+    "# ib.show(query1_data, visualize_data=True, include_window_info=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6faa7df8",
+   "metadata": {},
+   "source": [
+    "To materialize the PCollection into a pandas DataFrame object, we can use `collect`."
```

Comment:
Maybe add a link to pandas DataFrame object? I have no idea what is that.

Reply:
Got it, adding https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe as the link.